src/super_gradients/training/utils/predict/prediction_results.py

```diff
@@ -41,6 +42,70 @@ def save(self, *args, **kwargs) -> None:
         pass
 
 
+@dataclass
+class ImageClassificationPrediction(ImagePrediction):
+    """Object wrapping an image and a classification model's prediction.
+
+    :attr image:        Input image
+    :attr predictions:  Predictions of the model
+    :attr class_names:  List of the class names to predict
+    """
+
+    image: np.ndarray
+    prediction: ClassificationPrediction
+    class_names: List[str]
+
+    def draw(self, show_confidence: bool = True) -> np.ndarray:
```

Comment:
I suggest to move the actual implementation of "draw label overlay on top of the image" to regular function similar to how `draw_bbox` is made:
`draw_image_with_header(image, text, text_color) -> np.ndimage`
And use it here. 
Pros: If we at some point want to reuse the drawing funciton we would simply call that function.

Reply:
Thanks. ill fix the black formatter (its still a draft).